recipe/meta.yaml

```diff
@@ -49,11 +52,20 @@ test:
     - if not %MSMPI_BIN% == %LIBRARY_BIN% exit 1  # [win]
     - if not %MSMPI_INC% == %LIBRARY_INC% exit 1  # [win]
     - if not %MSMPI_LIB64% == %LIBRARY_LIB% exit 1  # [win]
+
+    # check if binaries work ok
     - where mpiexec.exe  # [win]
     - where smpd.exe  # [win]
     - mpiexec.exe -d 3 -n 2 Tests\\MpiPingpong.exe -m  # [win]
     - mpiexec.exe -d 3 -n 2 Tests\\IMB_MPI1.exe PingPong PingPing PingPongAnySource PingPingAnySource Sendrecv Allreduce -off_cache -1 -msglog 10  # [win]
-    # TODO(leofang): add compilation tests, at least test compiling and running mpi4py
+
+    # test compiling + linking + execution; also check if MSMPI_VER is patched correctly
+    - git clone https://github.com/mpi4py/mpi4py.git
+    - cd mpi4py
+    - {{ PYTHON }} -m pip install --no-deps . -vv
+    - python -c "from mpi4py import MPI; print(MPI.Get_library_version())"
```

Comment:
I would also like to see the output of `MPI.get_vendor()`. That one depends on the `MSMPI_VER` define.

Reply:
Yes, let's do this in mpi4py-feedstock.